fof/View/DataView/Json.php

```diff
@@ -281,6 +281,16 @@ protected function onBeforeRead($tpl = null)
 	}
 
 	/**
+	 * The event which runs when we are displaying a single item JSON view
+	 *
+	 * @param   string  $tpl  The view sub-template to use
+	 */
+	protected function onAfterSave($tpl = null)
+	{
+		self::onBeforeRead($tpl);
```

Comment:
Because i want to display something **after** saving, and what i need is exactly the onBeforeRead method.
Should i move the onBeforeRead method in a protected function called "printItem" or something and call that from both events?